src/ontodia/data/utils.ts

```diff
@@ -0,0 +1,27 @@
+import * as N3 from "n3";
+import { RdfNode, Triple } from './sparql/sparqlModels';
+
+export function N3toRdfNode(entity: string): RdfNode {
```

Comment:
Please, rename this function so it starts with lowercase character, e.g. `n3toRdfNode`.